packages/data-context/src/sources/ProjectDataSource.ts

```diff
@@ -478,6 +478,18 @@ export class ProjectDataSource {
     return preferences[projectTitle] ?? null
   }
 
+  async getCodeGenGlobs () {
+    assert(this.ctx.currentProject, `Cannot find glob without currentProject.`)
+
+    const looseComponentGlob = '*.{js,jsx,ts,tsx,.vue}'
```

Comment:
Looks like there's an extra period for vue here:

```suggestion
    const looseComponentGlob = '*.{js,jsx,ts,tsx,vue}'
```